connector-packager/connector_packager/jar_jdk_packager.py

```diff
@@ -43,6 +43,9 @@ def get_min_support_version(file_list: List[ConnectorFile]) -> str:
         if connector_file.file_type == "connection-fields":
             min_version_tableau = "2020.3"
             reasons.append("Connector uses Connection Dialogs V2, which was added in the 2020.3 release")
+        elif connector_file.file_type == "oauth-config":
```

Comment:
You'll want the newest check first so we end up computing the max min_version_tableau.

Reply:
This does end up correctly stamping 2021.1 (the test checks for this), but the reason it does it the connection fields file is checked first (since it's first in the manifest), so it's kind of by accident. I think only changing the min_version if it's greater than the current one is probably safest.